app/actor/UserActor.scala

```diff
@@ -1,13 +1,15 @@
 package actor
 
 import akka.actor._
+import models.Message
+import utillities.Json
 
 object UserActor {
   def props: Props = Props(new UserActor)
 }
 
 class UserActor extends Actor {
   override def receive: Receive = {
-    case msg: String => ChatSystem.lounge ! msg
+    case jsonstring: String => ChatSystem.lounge ! Json.toObject[Message](jsonstring)
```

Comment:
Json.toObject[Message](jsonstring) is an `Option[Message]` you need to `.get` to get Message